app/decorators/decidim/admin/users_controller_decorator.rb

```diff
@@ -19,8 +20,76 @@ def user
     @user ||= original_collection.find(params[:id])
   end
 
+  def show
+    locale = params[:locale] || "ca"
+    @user ||= original_collection.find(params[:id])
+    @spaces = []
+    @user.participatory_processes.each do |process|
+
+      if process.participatory_process_group then
+        if process.participatory_process_group&.name[locale] != '' then
+          type = process.participatory_process_group&.name[locale]
+        else 
+          type = process.participatory_process_group&.name["ca"]
+        end
+      else 
+        type = t("models.user.fields.process_type", scope: "decidim.admin")
+      end
+
+      process_title = process.title[locale]
+      if process_title == '' then
+        process_title = process.title["ca"]
+      end
```

Comment:
why this kind of logic?
We're lying to the user making her believe that the process has title in the current locale when it is not true..
Is this a customer request?
The same for the next similar cases

Reply:
I just followed the current logic, if the title is not able in current language it shows it in catalan (default language), you can check that logic is the same in production